libscript/src/script-validate.hpp

```diff
@@ -0,0 +1,299 @@
+/* Copyright (C) 2003-2016 LiveCode Ltd.
+ 
+ This file is part of LiveCode.
+ 
+ LiveCode is free software; you can redistribute it and/or modify it under
+ the terms of the GNU General Public License v3 as published by the Free
+ Software Foundation.
+ 
+ LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+ WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ for more details.
+ 
+ You should have received a copy of the GNU General Public License
+ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+#include <libscript/script.h>
+
+#include "script-private.h"
+
+class MCScriptValidateContext
+{
+public:
+	// Constructors
+	//
+	MCScriptValidateContext(MCScriptModuleRef module,
+							MCScriptHandlerDefinition *definition,
+							uindex_t address,
+							uindex_t *arguments,
+							uindex_t argument_count);
```

Comment:
Probably being daft, but I can't seem to find the implementation of this constructor anywhere in this patch.